source/languageHandler.py

```diff
@@ -137,27 +137,34 @@ def pgettext(context, message):
 			return unicode(message)
 	return pgettext
 
+def getWindowsLanguage():
+	"""
+	Fetches the locale name of the user's configured language in Windows.
+	"""
+	windowsLCID=ctypes.windll.kernel32.GetUserDefaultUILanguage()

```

Comment:
Now we are no longer required to support Windows XP, I'd like to suggest using [GetUserPreferredUILanguages](https://msdn.microsoft.com/en-us/library/windows/desktop/dd318139(v=vs.85).aspx)